src/containers/CategoriesContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { useSelector } from 'react-redux';
+
+import Category from '../components/Category';
+
+export default function CategoriesContainer() {
+  const { categories } = useSelector((state) => ({
+    categories: state.category.categories,
+  }));
+
+  function handleClick() {
+    // TODO
+  }
+
+  return (
+    <ul>
+      {categories.map((category) => (
+        <Category
+          key={category.id}
+          name={category.name}
+          onClick={handleClick}
+        />
```

Comment:
```suggestion
      {categories.map(({ id, name }) => (
        <Category
          key={id}
          name={name}
          onClick={handleClick}
        />
```

이 부분은 구조 분해 할당으로 처리해줘도 의도가 전달 될 것 같네요~